SDK/AppCenterRum/Microsoft.AppCenter.Rum/Microsoft.AppCenter.Rum.csproj

```diff
@@ -14,7 +14,7 @@
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Microsoft.AppCenter.Rum.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.1.0" PrivateAssets="All" />
+    <PackageReference Include="MSBuild.Sdk.Extras"/>
```

Comment:
Invalid change. Please read [the package readme](https://github.com/onovotny/MSBuildSdkExtras/blob/master/README.md)